### ISSUE TITLE
Not getting match.

### ISSUE THREAD

**Issue by zozo6015** [Yelp/elastalert#1579]
Hello, 

I have set up elastalert on elasticsearch 6.2.2
I am using any type with the following configuration:

    es_host: localhost 
    es_port: 9200
    _source_enabled: true
    name: Log checker for SEVERE in Jetty
    type: any
    index: filebeat-*
    timeframe:
      minutes: 5
    filter:
    - term:
        system.jetty.severity: "SEVERE"
    aggregation:
      minutes: 5
    aggregation_key: '@environment'
    alert:
    - "email"
    - "slack"
    slack_webhook_url: webhook url 
    slack_username_override: VC-alerts
    slack_msg_color: 'danger'
    email:
    - "email@example.com"

running elastalert-test-rules does not give any match however running the following query in elasticsearch I get the some results:

    curl -XGET "http://localhost:9200/filebeat-*/_search" -H 'Content-Type: application/json' -d'
    {
      "query": {
        "match": {
          "system.jetty.severity": {
            "query": "SEVERE"
          }
        }
      }
    }'

Any idea?

**Comment by zozo6015** [comment]
Any idea?

**Comment by bat9r** [comment]
I think you have same trouble https://github.com/Yelp/elastalert/issues/1449

**Comment by zozo6015** [comment]
Nope. even not running in test mode I don't get any match which is not correct because I should get lots of matches in 15 minutes timestamp.
I guess that the issue is coming from not having python elasticsearch module for version 6.2.2. Who knows when that will be released.

**Comment by adolia** [comment]
Hi there!
I have the same issue, not getting matches for all rules using Elasticsearch 6.2.2.
Any idea what is the issue here?
Thanks in advance.
